src/batch_norm.cpp

```diff
@@ -565,19 +570,52 @@ void bnBwdTrainSelectSingle(const Handle& handle,
                             double epsilon,
                             ConstData_t savedMean,
                             ConstData_t savedInvVariance,
-                            float inhw)
```

Comment:
I see you are calculating inverse nhw in OCL code. is it possible to send it inhw directly?